exec/java-exec/src/main/codegen/templates/MapWriters.java

```diff
@@ -52,9 +52,18 @@
   private final Map<String, FieldWriter> fields = Maps.newHashMap();
   <#if mode == "Repeated">private int currentChildIndex = 0;</#if>
 
-  public ${mode}MapWriter(${containerClass} container, FieldWriter parent) {
+  private final boolean unionEnabled;
+  private final boolean unionInternalMap;
```

Comment:
Can you give a description of the unionInternalMap parameter and why that is needed in a javadoc?